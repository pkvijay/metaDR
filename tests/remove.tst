remove BADPATH
create /test 123
remove /test
remove /test
remove /foo
create /test abc
create /test/b def
remove /test
