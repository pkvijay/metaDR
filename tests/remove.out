MALFORMED KEY
CREATED
REMOVED
KEY NOT FOUND
KEY NOT FOUND
CREATED
CREATED
HAS CHILDREN
